clicker/src/components/Clicker-class.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from "react";
+
+class Clicker2 extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            counter: 0,
+        };
+    }
+
+    plus = () => {
+        this.setState({
+            counter: this.state.counter += 1,
```

Comment:
Do not ever mutate the state object!
```suggestion
            counter: this.state.counter + 1,
```